### ISSUE TITLE
Multiline in an if statement fails with an `indent` issue

### ISSUE THREAD

**Issue by ZakTaccardi** [pinterest/ktlint#900]
## Expected Behavior
I expect the following to pass formatting.

```kotlin
fun broken(key: String): Letter {
    for (letter in Letter.values()) {
        if (
            letter.value
                .equals(
                    key,
                    ignoreCase = true
                )
        ) {
            return letter
        }
    }
    return Letter.B
}
```

## Observed Behavior
Fails with: `Unexpected indentation (28) (should be 32) (indent)`

## Steps to Reproduce
```kotlin
enum class Letter(val value: String) {
    A("a"),
    B("b");
}
fun broken(key: String): Letter {
    for (letter in Letter.values()) {
        if (
            letter.value
                .equals(
                    key,
                    ignoreCase = true
                )
        ) {
            return letter
        }
    }
    return Letter.B
}
fun fixed1(key: String): Letter {
    for (letter in Letter.values()) {
        if (
            letter.value
                .equals(key, ignoreCase = true)
        ) {
            return letter
        }
    }
    return Letter.B
}
fun fixed2(key: String): Letter {
    for (letter in Letter.values()) {
        if (letter.value.equals(key, ignoreCase = true)) {
            return letter
        }
    }
    return Letter.B
}
```

## Your Environment
* Version of ktlint used: `0.38.1` (command line)
* AS `4.0.1` with AGP `3.6`
* Operating System and version: MacOS 10.15.6

**Comment by ZakTaccardi** [comment]
This could likely be the same issue and have the same resolution as #898

**Comment by Tapchicoma** [comment]
yes, it fails on current `master`
